packages/fxa-auth-server/fxa-oauth-server/scripts/retire-old-signing-key.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*
+
+ Usage:
+ scripts/retire-new-signing-key.js
+
+ Will zero out the old signing key in ./config/oldKey.json.
+
+ */
+
+//eslint-disable no-console
+
+const fs = require('fs');
+const config = require('../lib/config');
+
+// We don't use this, but we load it to check key config.
+require('../lib/keys');
```

Comment:
I'll admit I'm not a fan of relying on import side-effects. Why not have a key verification function that can be called?

This could be done as a follow-up PR though, I don't want to hold this one up anymore.

Reply:
Thanks, yeah, I know what you mean. The competing goal here is to make it very very hard to run the server in a broken configuration, but that might be better achieved by having the server process call a special `validateKeys()` function before starting up.

I'll timebox a brief attempt at such a refactor, as a replacement for the weird "no really I want you to start up with an incomplete config" environment var that's been added here. But I won't block on it if it turns into a rabbit-hole.

Reply:
> But I won't block on it if it turns into a rabbit-hole.

I poked at this but ran out of steam pretty quick, let's leave it to a future followup...